docs/basics/beginners-contributing-guide.md

```diff
@@ -0,0 +1,119 @@
+# Beginners Contributing Guide
+
+## Prerequisites:
+* Docker is already running in your environment
+* You have a [Github](https://github.com) account and are logged in
+* This guide was written for linux
+
+__Note__: **This document will use variables thought out make sure to change them as needed.**
+
+### 1. Install Git
+This section will install Git.
+
+0. Log in to the server you are going to use for Git.
+
+0. Type `add-apt-repository ppa:git-core/ppa`.
+
+0. Type `apt-get install git` git will be installed.
+
+0. Type `git config --global user.email "email@example.com"` to set an email address that will be used when you post to GitHub.
+
+0. Type `git config --global user.email` to verify the email address.
+
```

Comment:
Don't forget to set your name also:
```
git config --global user.name "John Doe"
```
You can also review all your configurations with `git config --list`

Reply:
thanks @py-crash i added the username command but left the other one out as its good info but its not needed to post.

Reply:
Don't forget to change `git config --global user.email` for `git config --list` so the user checks his Name and Email in the same command